.vscode/snippets.code-snippets

```diff
@@ -18,6 +18,29 @@
             " limitations under the License.",
```

Comment:
Shouldn't we put the actual license in here?

Reply:
;) It's there, you're just not seeing it in the GitHub UI since it hasn't changed ;).